docs/howto/determining-packages.md

```diff
@@ -0,0 +1,172 @@
+# Determining Packages in Pangeo Docker Images
+
+For deployments of Pangeo Docker Images using JupyterHub or Binder it can be sometimes unclear to the user exactly which image and tag is being used. Here are the images available as Pangeo Docker Images:
+
+| Image           | Description                                   | 
+|-----------------|-----------------------------------------------|
+| [base-notebook](base-notebook/packages.txt) | minimally functional image for pangeo hubs | 
+| [pangeo-notebook](pangeo-notebook/packages.txt) | base-notebook + core earth science analysis packages |
+| [pytorch-notebook](pytorch-notebook/packages.txt) | pangeo-notebook + GPU-enabled pytorch |
+| [ml-notebook](ml-notebook/packages.txt) | pangeo-notebook + GPU-enabled tensorflow2 | 
+| [forge](ml-notebook/packages.txt) | pangeo-notebook + [Apache Beam](https://beam.apache.org/) support| 
+
+Each image is also assigned a tag of the form `YYYY.MM.DD` indicating when it was last updated.
+
+*Click on the image name in the table above for a current list of installed packages and versions for the most recent tag of each image*
+
+## How to find my current image tag
+
+The environment variable `JUPYTER_IMAGE_SPEC` will tell you what Pangeo Docker Image is currently in use.
```

Comment:
My suggestion here is to show people with how to access `JUPYTER_IMAGE_SPEC` (`echo $JUPYTER_IMAGE_SPEC` in a terminal or even just `!echo $JUPYTER_IMAGE_SPEC` in a notebook), and then *describe* the various parts there. For example, the value may actually be of three forms:

1. `docker.io/pangeo-data/pangeo-notebook:tag`
2. `pangeo-data/pangeo-notebook:tag`
3. `quay.io/pangeo-data/pangeo-notebook:tag`

I think describing the registry (`docker.io`, which is often omitted, to mean dockerhub or `quay.io`), the repository (`pangeo-data/pangeo-notebook`) and the `tag` is cleaner and more helpful to the target audience I think.